app/helpers/navigation_helper.rb

```diff
@@ -1,5 +1,6 @@
 module NavigationHelper
   RETURN_TO_TOP_PAGES = ['show', 'metrics', 'introduction', 'overview', 'implementation']
+  PRACTICE_EDITOR_PAGES = ['introduction', 'editors', 'overview', 'contact', 'implementation', 'adoptions', 'about']
```

Comment:
I think we can remove the contact string here, since we combined the contact and about pages into just the about page